modifiers/multi.go

```diff
@@ -125,7 +125,25 @@ func setValue(ctx context.Context, fl mold.FieldLevel) error {
 		fl.Field().Set(reflect.MakeChan(fl.Field().Type(), buffer))
 
 	case reflect.Ptr:
+		// Handle pointer fields by:
+		// 1. Creating a new pointer point to empty value with reflect.New()
 		fl.Field().Set(reflect.New(fl.Field().Type().Elem()))
+
+		// 2. Attempting to set its underlying value
+		// Try to convert the parameter string to the appropriate primitive type
+		// that the pointer references (e.g., *string, *int, *bool)
+		value, err := mold.GetPrimitiveValue(fl.Field().Type().Elem().Kind(), fl.Param())
```

Comment:
Instead of the new `GetPrimitiveValue` function could we change the function signature of `setValue` to accept `reflect.Value` and `string(the param)` and recursively call it dereferencing the pointer.

This would avoid needing the new function I think and addition of the new error types as well.

Reply:
I found some time to do and test this today https://github.com/go-playground/mold/pull/49

everything should be working as expected as of Release 4.5.1 https://github.com/go-playground/mold/releases/tag/v4.5.1